scripts/agent-trader/agent_trader.py

```
@@ -196,9 +214,35 @@ def summary():
     if closed:
         print(f"record: {wins}-{len(closed)-wins}  hit_rate: {wins/len(closed):.2%}")
         print(f"P&L net: ${pnl:+.2f}  bankroll: ${BANKROLL0 + pnl:.2f}  Brier: {brier}")
+    hr = _honest_record(bets)
+    if hr and hr["n_pending"]:
+        print(f"mark-to-market: {hr['wins']}-{hr['losses']}  P&L ${hr['pnl']:+.2f}  "
+              f"bankroll: ${hr['bankroll']:.2f}  "
+              f"({hr['n_pending']} open position(s) already decided by the market)")
     print(f"capital at risk (open): ${sum(b['stake'] for b in openb):.0f}")
 
 
+def _honest_record(bets):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_honest_record`

(ANN202)